### PR TITLE
Added Bold and Italic buttons

### PR DESCRIPTION
**Pull request by gpasents**
Added the buttons and images for bold and italic options for fonts

### CODE REVIEW

File: src/simplejavatexteditor/UI.java [hunk 1]
```diff
@@ -485,8 +504,30 @@ else if (e.getSource() == openFile || e.getSource() == openButton) {
...
+            }
+            
+        }// If the source of the event was the "Italic" button
+        else if (e.getSource() == italicButton) {
+            if(textArea.getFont().getStyle() == Font.ITALIC){
+                textArea.setFont(textArea.getFont().deriveFont(Font.PLAIN));
+                
+            }
+            else{
+
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Extra line here ;)

File: src/simplejavatexteditor/UI.java [hunk 2]
```diff
@@ -485,8 +504,30 @@ else if (e.getSource() == openFile || e.getSource() == openButton) {
...
+                
+            }
+            else{
+                
+                textArea.setFont(textArea.getFont().deriveFont(Font.ITALIC));
+            }
+            
         }
-
+
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Extra spaces here

File: src/simplejavatexteditor/UI.java [hunk 3]
```diff
@@ -485,8 +504,30 @@ else if (e.getSource() == openFile || e.getSource() == openButton) {
...
+            else{
+                
+                textArea.setFont(textArea.getFont().deriveFont(Font.BOLD));
+            }
+            
+        }// If the source of the event was the "Italic" button
+        else if (e.getSource() == italicButton) {
+            if(textArea.getFont().getStyle() == Font.ITALIC){
+                textArea.setFont(textArea.getFont().deriveFont(Font.PLAIN));
+
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Extra line here

File: src/simplejavatexteditor/UI.java [hunk 4]
```diff
@@ -485,8 +504,30 @@ else if (e.getSource() == openFile || e.getSource() == openButton) {
...
         else if (e.getSource() == saveFile || e.getSource() == saveButton) {
             saveFile();
+        }// If the source of the event was the "Bold" button
+        else if (e.getSource() == boldButton) {
+            if(textArea.getFont().getStyle() == Font.BOLD){
+                textArea.setFont(textArea.getFont().deriveFont(Font.PLAIN));
+                
+            }
+            else{
+
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Extra line here

File: src/simplejavatexteditor/UI.java [hunk 5]
```diff
@@ -485,8 +504,30 @@ else if (e.getSource() == openFile || e.getSource() == openButton) {
         } // If the source of the event was the "save" option
         else if (e.getSource() == saveFile || e.getSource() == saveButton) {
             saveFile();
+        }// If the source of the event was the "Bold" button
+        else if (e.getSource() == boldButton) {
+            if(textArea.getFont().getStyle() == Font.BOLD){
+                textArea.setFont(textArea.getFont().deriveFont(Font.PLAIN));
+
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Extra line here

File: src/simplejavatexteditor/UI.java [hunk 6]
```diff
@@ -485,8 +504,30 @@ else if (e.getSource() == openFile || e.getSource() == openButton) {
         } // If the source of the event was the "save" option
         else if (e.getSource() == saveFile || e.getSource() == saveButton) {
             saveFile();
+        }// If the source of the event was the "Bold" button
+        else if (e.getSource() == boldButton) {
+            if(textArea.getFont().getStyle() == Font.BOLD){
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Could you add a space after `if` to be `if (textArea.getFont()`

File: src/simplejavatexteditor/UI.java [hunk 7]
```diff
@@ -485,8 +504,30 @@ else if (e.getSource() == openFile || e.getSource() == openButton) {
...
+                textArea.setFont(textArea.getFont().deriveFont(Font.PLAIN));
+                
+            }
+            else{
+                
+                textArea.setFont(textArea.getFont().deriveFont(Font.BOLD));
+            }
+
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Extra line here

File: src/simplejavatexteditor/UI.java [hunk 8]
```diff
@@ -485,8 +504,30 @@ else if (e.getSource() == openFile || e.getSource() == openButton) {
...
+        else if (e.getSource() == italicButton) {
+            if(textArea.getFont().getStyle() == Font.ITALIC){
+                textArea.setFont(textArea.getFont().deriveFont(Font.PLAIN));
+                
+            }
+            else{
+                
+                textArea.setFont(textArea.getFont().deriveFont(Font.ITALIC));
+            }
+
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Unnecessary extra line here

**Review by pH-7 — CHANGES_REQUESTED** [review]

File: src/simplejavatexteditor/UI.java [hunk 9]
```diff
@@ -88,7 +96,7 @@
     public UI() {
 
         // Set the initial size of the window
-        setSize(700, 500);
+        setSize(800, 500);
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java:99]: Is there a reason why you change the size..? Didn't it fit with the changes..?

**Comment by pH-7** [comment]
Hi @gpasents 

Thanks a lot for contributing! Highly appreciate! :smiley:

Please see my reviews.

File: src/simplejavatexteditor/UI.java [hunk 10]
```diff
@@ -55,9 +58,14 @@
     private final JMenuItem newFile, openFile, saveFile, close, cut, copy, paste, clearFile, selectAll, quickFind,
             aboutMe, aboutSoftware, wordWrap;
     private final JToolBar mainToolbar;
-    JButton newButton, openButton, saveButton, clearButton, quickButton, aboutMeButton, aboutButton, closeButton;
+    JButton newButton, openButton, saveButton, clearButton, quickButton, aboutMeButton, aboutButton, closeButton,boldButton,italicButton;
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Can you add a space after the comma to be as `closeButton, boldButton, italicButton;`

**Review by pH-7 — CHANGES_REQUESTED** [review]

File: src/simplejavatexteditor/UI.java (hunk 9 above)
Comment (gpasents) [reply src/simplejavatexteditor/UI.java:99]: The new buttons make the font size area dissapear, so i increased the window's size

**Review by pH-7 — COMMENTED** [review]
👍

**Comment by pH-7** [comment]
Great! Thanks alot @gpasents and see you hopefully another time. Take care! 😃

**Pull request merged**